causal_testing/generation/abstract_causal_test_case.py

```diff
@@ -29,7 +30,7 @@ def __init__(
         effect_modifiers: set[Variable] = None,
         estimate_type: str = "ate",
     ):
-        assert treatment_variables.issubset(scenario.variables.values()), (
+        assert {treatment_variables}.issubset(scenario.variables.values()), (
```

Comment:
Could we not just do `in` instead of making a singleton set?

Reply:
Also we could maybe call it `treatment_variable` rather than `treatment_variables` plural if there's only one...

Reply:
Thanks Michael, that's a lot nicer implementation.

Reply:
Also I've got another branch going to clean up all the multiple variable refactors so it will done in there :)